mods/common/languages/en.ftl

```diff
@@ -130,7 +130,7 @@ complete = { $percentage }% complete
 ## LobbyLogic, InGameChatLogic
 chat-availability =
     { $seconds ->
-        [zero] Chat Disabled
+        [0] Chat Disabled
         [one] Chat available in { $seconds } second...
```

Comment:
This case seems different. We use [one] here, so to keep it consistent should we change to [1]?

Reply:
We shouldn't change it in this case - "one" is a category and there are languages that don't care about it.

Reply:
And using `[0]` is also not correct. "Chan Disabled" should be a separate message because it has nothing to do with the number parameter - it just happens so that in our logic "0" means chat disabled.

Reply:
Addressing this is better suited for one of the follow-up PRs though.